src/status_im/contexts/wallet/events.cljs

```diff
@@ -386,3 +389,41 @@
             {:title   title
              :subject title
              :message content})]]}))
+
+(rf/reg-event-fx
+ :wallet/blockchain-status-changed
+ (fn [{:keys [db]} [{:keys [message]}]]
+   (let [chains                  (-> (transforms/json->clj message)
+                                     (update-keys (comp utils.number/parse-int name)))
+         down-chains             (-> (select-keys chains
+                                                  (for [[k v] chains :when (= v "down")] k))
+                                     keys)
+         test-networks-enabled?  (get-in db [:profile/profile :test-networks-enabled?])
+         filter-fn               (fn [chain-id]
+                                   (let [mainnet-chain? (contains? constants/mainnet-chain-ids chain-id)]
+                                     (if test-networks-enabled?
+                                       mainnet-chain?
+                                       (not mainnet-chain?))))
+         chains-filtered-by-mode (remove filter-fn down-chains)
+         chains-down?            (seq chains-filtered-by-mode)
+         chain-names             (when chains-down?
+                                   (->> (map #(-> (utils/id->network %)
+                                                  name
+                                                  string/capitalize)
+                                             chains-filtered-by-mode)
+                                        set
+                                        (string/join ", ")))]
```

Comment:
I think that `set` is not needed, actually, sets have no order, so this might lead to an inconsistent resulting string

Reply:
There's a `distinct` clojure function :)

Reply:
Ah yes! 😄 Totally forgot that. I will update. 👍  Thanks!